### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by slipher**

### CODE REVIEW

File: Settings.qml [hunk 1]
```diff
@@ -45,7 +45,8 @@ Item {
             }
             TextField {
                 text: updaterSettings.commandLine
-                onAccepted: updaterSettings.commandLine = this.text
+                onEditingFinished: updaterSettings.commandLine = this.text
+                validator: RegExpValidator { regExp: /.*%command%.*/ }
```
Comment (DolceTriade) [inline Settings.qml]: This isn't super good UX. Maybe do the validation in onEditingFinished?

File: unix.cpp [hunk 2]
```diff
@@ -61,6 +62,14 @@ QString defaultInstallPath()
     return QStandardPaths::writableLocation(QStandardPaths::GenericDataLocation) + "/unvanquished/base";
 }
 
+bool badRootInstallLocation(const QString& installPath)
```
Comment (DolceTriade) [inline unix.cpp]: validateInstallPath?

**Review by DolceTriade — CHANGES_REQUESTED** [review]

File: unix.cpp (hunk 2 above)
Comment (slipher) [reply unix.cpp]: Done.

File: Settings.qml (hunk 1 above)
Comment (slipher) [reply Settings.qml]: Well, I don't want to spend a lot of time on this now. I can remove the validator if you think that's better.

Comment (DolceTriade) [reply Settings.qml]: I think we can trivially make this better by assuming that if `%command%` isn't there, then the arguments that are saved should be appendeded to `%command%`

**Review by DolceTriade — APPROVED** [review]

**Comment by DolceTriade** [comment]
But I don't feel strongly enough to block this change on that.

File: Settings.qml (hunk 1 above)
Comment (slipher) [reply Settings.qml]: Good idea, done.

**Pull request merged**